keepass-mode.el

```diff
@@ -98,9 +114,9 @@
 (add-to-list 'auto-mode-alist '("\\.kdbx\\'" . keepass-mode))
 (add-to-list 'auto-mode-alist '("\\.kdb\\'" . keepass-mode))
 
-(defun keepass-mode-get-password (entry)
-  "Retrieve password for ENTRY."
-  (keepass-mode-get-field "Password" (shell-command-to-string (keepass-mode-command (keepass-mode-quote-unless-empty entry) "show -s"))))
+(defun keepass-mode-get (field entry)
```

Comment:
This will break the [tests](https://github.com/ifosch/keepass-mode/blob/master/tests/test-keepass-mode.el#L35), which should also include the new cases.